transformer_engine/common/include/transformer_engine/rmsnorm.h

```diff
@@ -18,6 +18,15 @@ extern "C" {
 #endif
 
 /*! \brief Compute RMSNorm on the input.
+ *
+ * The formula used:
+ * @f[
+ * y = \frac{x}{RMS_\varepsilon(x)}\gamma
+ * @f]
+ * where
+ * @f[
+ * RMS_\varepsilon(x) = \sqrt{\frac{1}{n}\sum_{i=0}^nx_i^2 + \varepsilon}
```

Comment:
Pedantic:
```suggestion
 * RMS_\varepsilon(x) = \sqrt{\frac{1}{n}\sum_{i=1}^n x_i^2 + \varepsilon}
```
Alternatively:
```c++
 * RMS_\varepsilon(x) = \sqrt{\frac{1}{n}\sum_{i=0}^{n-1} x_i^2 + \varepsilon}
```